tests/python/unittest/test_tvmscript_roundtrip.py

```diff
@@ -2888,6 +2888,67 @@ def test_opaque_block():
     assert len(root_block.body.body[1].block.iter_vars) == 0
 
 
+@tvm.script.tir
+def rank0(a: ty.handle) -> None:
+    A = tir.match_buffer(a, (), "float32")
+    B = tir.alloc_buffer((), "float32")
+    A[()] = 2
+    B[()] = A[()]
+
+
+def test_rank0_buffers():
+    func = rank0
+    rt_func = tvm.script.from_source(tvm.script.asscript(func, True))
+    tvm.ir.assert_structural_equal(func, rt_func)
+
+
+@tvm.script.tir
+def rank0_block(a: ty.handle) -> None:
+    A = tir.match_buffer(a, (), "float32")
+    B = tir.alloc_buffer((), "float32")
+    tir.store(B.data, 0, tir.load("float32", A.data, 0))
+
+    with tir.block([], "update") as []:
+        tir.reads([A[()]])
+        tir.writes([B[()]])
+        for i in range(0, 1):
+            B[()] = A[()]
+
+
+def test_rank0_blocks():
+    func = rank0_block
+    print(tvm.script.asscript(func, True))
```

Comment:
Is it necessary to print here?
Perhaps you want to check the output with string and assert?

Reply:
Totally forgot to remove the print statements. Done.